src/app/clusters/dishwasher-alarm-server/dishwasher-alarm-server.cpp

```diff
@@ -355,7 +354,7 @@ static Status ResetHandler(const app::ConcreteCommandPath & commandPath, const B
     if (!DishwasherAlarmServer::Instance().HasResetFeature(endpoint))
     {
         ChipLogProgress(Zcl, "Dishwasher Alarm feature: Unsupport Reset Command");
-        return app::ToInteractionModelStatus(EMBER_ZCL_STATUS_UNSUPPORTED_COMMAND);
+        return EMBER_ZCL_STATUS_UNSUPPORTED_COMMAND;
```

Comment:
Should this be changed to an interaction model status?

Reply:
We should always prefer IM status to `EMBER_ZCL_xxx` constants. I think the issue here is that the original code should have said `Status::UnsupportedCommand` directly rather than `app::ToInteractionModelStatus.

Strongly recommend replacing every usage of `app::ToInteractionModelStatus(EMBER_ZCL_*)` to plaing InteractionModel::Status enums

Reply:
that is indeed the goal and will be done in a follow PR when the complete `af-enums.h` file is delete from the SDK 

Still need PR #32055 and PR #32056 to merge before doing so.